frame/currency-factory/README.md

```diff
@@ -1,41 +1,51 @@
 # Overview
 
-Currency pallet allows to create new sovereign(local consensus) currencies(assets) from thin air.
+This pallet creates new sovereign(local consensus) currencies(assets) from thin air.
 
-Each currency falls into one of next categories:
+Each currency falls into one of the following categories:
 
 - new currency issued by governance
-- currency produced as consequences of execution of protocol and depends on some base currency(ies), usually named LP
-- currency mapping remote location of asset to local identifiers, foreign currency
+- currency produced as consequences of a protocol execution that depends on one or more base currencies, usually named LP
+- currency mapping the remote location of an asset to local identifiers associated with the foreign currency.
+
+
+Adding well-known identifiers for assets occurs under the following conditions:
+
+- baked into the runtime during compilation
+- added as part of genesis chain specification
+- created in runtime
+
+Not registered in this pallet are compilation time currencies.
   
 ## Basics
 
-Currency must be identified, so factory allows to produce new identifiers.
+We must identify currency. This pallet allows the production of new identifiers to fill that requirement.
 
-Currency to be useful must have amounts. Amounts must be stored on accounts. Accounts must pay ED to prevent spam.
-So each currency must have ED defined.
+A Currency needs an associated amount stored on accounts. Said accounts must pay ED(Existential Deposit) to prevent spam.
+So each currency must have an [ED](../../docs/0002-rent-deposit.md) defined.
```

Comment:
```suggestion
A Currency needs an associated amount stored on accounts. To prevent spam, accounts are required to pay an ED (Existential Deposit). As such, each currency must have an [ED](../../docs/0002-rent-deposit.md) defined.
```

Reply:
Unsure if "A Currency needs an associated amount stored on accounts." can be removed